### ISSUE TITLE
Running NOverlap after ForceLink is finised

### ISSUE THREAD

**Issue by daphil19** [dunnock/react-sigma#34]
Greetings!

It's a pretty common issue in sigma that thinks like ForceAtlas2 and ForceLink can cause a lot of node overlap (i.e. sigma issues [613](https://github.com/jacomyal/sigma.js/issues/613), [107](https://github.com/jacomyal/sigma.js/issues/107), and  [687](https://github.com/jacomyal/sigma.js/issues/687)).

The approach I'm trying to take (and the one I think makes the most sense) is to perform ForceLink and then remove the overlap once that's complete. I've tried composing both `<ForceLink/>` and `<NOverlap/>` on the same level, but doing that has `NOverlap` mount and run immediately (which is before `ForceLink` is done). I've also tried comopsing it inside of `ForceLink`, but that results in an error.

Is it possible for me to force `NOverlap` to run once `ForceLink` is done using some composition? Or is that something that I'll have to handle in Sigma directly? Thanks!

**Comment by dunnock** [comment]
Hi, I have just added composition possibility to ForceLink, please confirm if it worked for you. Version 1.2.22.

**Comment by dunnock** [comment]
Actually version is 1.2.23

**Comment by daphil19** [comment]
@dunnock doesn't look like 1.2.23 did it for me...

Here's the error that I'm getting 
```
Uncaught DOMException: Failed to execute 'postMessage' on 'Worker': Symbol(react.element) could not be cloned.
    at Supervisor.Sigma.17.Supervisor.sendByteArrayToWorker (webpack:///./~/react-sigma/sigma/layout.forceLink.js?:250:19)
    at Supervisor.Sigma.17.Supervisor.start (webpack:///./~/react-sigma/sigma/layout.forceLink.js?:263:12)
    at Object.Sigma.17.sigma.layouts.startForceLink (webpack:///./~/react-sigma/sigma/layout.forceLink.js?:444:16)
    at ForceLink._refreshGraph (webpack:///./~/react-sigma/lib/ForceLink.js?:139:21)
    at ForceLink.componentDidMount (webpack:///./~/react-sigma/lib/ForceLink.js?:98:12)
    at eval (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:264:25)
    at measureLifeCyclePerf (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:75:12)
    at eval (webpack:///./~/react-dom/lib/ReactCompositeComponent.js?:263:11)
    at CallbackQueue.notifyAll (webpack:///./~/react-dom/lib/CallbackQueue.js?:76:22)
    at ReactReconcileTransaction.close (webpack:///./~/react-dom/lib/ReactReconcileTransaction.js?:80:26)
```

**Comment by dunnock** [comment]
Yes, saw that, it should be fixed with 1.2.26

**Comment by daphil19** [comment]
Working now! Thanks!
